IPython/core/interactiveshell.py

```diff
@@ -334,15 +334,26 @@ def _exiter_default(self):
     filename = Unicode("<ipython console>")
     ipython_dir= Unicode('').tag(config=True) # Set to get_ipython_dir() in __init__
 
-    # Input splitter, to transform input line by line and detect when a block
-    # is ready to be executed.
-    input_splitter = Instance('IPython.core.inputsplitter.IPythonInputSplitter',
-                              (), {'line_input_checker': True})
+    # Used to transform cells before running them, and check whether code is complete
+    input_transformer_manager = Instance('IPython.core.inputtransformer2.TransformerManager',
+                                         ())
 
-    # This InputSplitter instance is used to transform completed cells before
-    # running them. It allows cell magics to contain blank lines.
-    input_transformer_manager = Instance('IPython.core.inputsplitter.IPythonInputSplitter',
-                                         (), {'line_input_checker': False})
+    @property
+    def input_transformers_cleanup(self):
+        return self.input_transformer_manager.cleanup_transforms
+
+    input_transformers_post = List([],
+        help="A list of string input transformers, to be applied after IPython's "
+             "own input transformations."
+    )
+
+    @property
+    def input_splitter(self):
+        """Make this available for compatibility
```

Comment:
Make this available for backward compatibility (pre-0.7 release) with existing code.

For example, ipykernel...

New code should use input_transformer_manager.

Reply:
@willingc Your  `7` and `0` keys are either swapped, or you're used to software below 1.0 :-) I've edited a couple of your comments to say `7.0` instead of `0.7` as I think that's what you meant.

Reply:
Yes, I had a backwards moment :wink: